test/PowerShellEditorServices.Test/Language/SymbolsServiceTests.cs

```diff
@@ -285,7 +285,7 @@ public void FindsOccurrencesOnClass()
         {
             IReadOnlyList<SymbolReference> occurrencesResult = GetOccurrences(FindsOccurrencesOnTypeSymbolsData.ClassSourceDetails);
             Assert.Equal(2, occurrencesResult.Count);
-            Assert.Equal("[SuperClass]", occurrencesResult[occurrencesResult.Count - 1].SymbolName);
+            Assert.Equal("SuperClass", occurrencesResult[occurrencesResult.Count - 1].SymbolName);
```

Comment:
Because you ordered `GetOccurrences()`. Don't see any code changed, so `TypeExpression` and `TypeConstraint` includes brackets while `TypeDefinition` (this) does not. :)

Reply:
I have no idea. Maybe the tests didn't pass after all (can't remember). This seems okay though. 👍